### DISCUSSION TITLE
Dynamic link to blog post

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/wn-blog-plugin#58] [Q&A] [answered]
Hi,

Help please get dynamic link to blog post, with slug "some-page-123", this slug on all languages for this post same.

For static pages plugin i use:
<a href="{{ 'some-page-123' | staticPage }}"</a>

And cms get link how i need with language prefix and it work fine.

How its look like for blog plugin where i create post with slug "some-page-123"

**Comment by mjauvin** [comment]
`{{ 'post-page' | page({slug:'some-page-123') }}`

**Comment by oim37** [comment]
I try but i add this - }, get an error, in general:
`<a href="{{ 'post-page' | page( {slug:'some-page-123'} ) }}"</a>` it open page without error, but link empty.

WitnerCMS last version.

**Comment by oim37** [comment]
I also try this example https://wintertricks.com/tricks/generate-a-link-to-a-cms-page
`{{ 'blog-post' | page({slug: 'slug-of-the-post'}) }}`
but it show empty link too

**Comment by mjauvin** [comment, marked as answer]
What's the file path for the blog post page ? You need to replace 'post-page' with that path. Example, if this is under "pages/post.htm", the file path is "post". If it's under "pages/blog/post.htm", the file path is "blog/post".

**Comment by oim37** [comment]
Now I understand, thanks for your help, I wouldn’t have guessed it in my case it, now it work good with all language prefixes.
